src/seedu/addressbook/data/person/Phone.java

```diff
@@ -37,23 +34,8 @@ public static boolean isValidPhone(String test) {
     }
 
     @Override
-    public String toString() {
-        return value;
-    }
-
-    @Override
-    public boolean equals(Object other) {
-        return other == this // short circuit if same object
-                || (other instanceof Phone // instanceof handles nulls
-                && this.value.equals(((Phone) other).value)); // state check
-    }
-
-    @Override
     public int hashCode() {
         return value.hashCode();
```

Comment:
hmm..the value variable does not exist in your `Phone` class already. Can this class file still compile?